### ISSUE TITLE
incorrect sorting of Germanic umlaut(e/s?)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [medavox/simply-do#6]
```
What steps will reproduce the problem?
1. add list (or an item to a list) that starts with the letters Ä, Ö or Ü, 
the Germanic umlaut(e/s?), see https://en.wikipedia.org/wiki/Germanic_umlaut
2. sorting the list
3. checking where the entry appears in the sorted list

What is the expected output? What do you see instead?
expected: Ä, Ö & Ü should be treated like A, O & U respectively during 
sorting
instead: The items appear at the list end. Which is very uncommon, therefore a 
bit irritating.

What version of the product are you using? On what operating system?
0.9.2 on CyanogenMod 9.10


```

Original issue reported on code.google.com by `cathryne...@gmail.com` on 6 Jul 2013 at 5:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.9.3. Locale specific sorting is now used.
```

Original comment by `keith.ki...@gmail.com` on 15 Aug 2014 at 11:49
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Merci :-)
```

Original comment by `cathryne...@gmail.com` on 16 Aug 2014 at 7:10
